lib/mail/fields/content_type_field.rb

```diff
@@ -144,10 +144,9 @@ def sanatize( val )
 
       # TODO: check if there are cases where whitespace is not a separator
       val = val.
+        gsub(/\s[\w\-]+:\s.*\\r\\n$/, ''). # cases with botched fields like Content-Type: text/html Content-Transfer-Encoding: 8bit\r\n
```

Comment:
We are processing all sorts of email. What is a mitigation for cases when we encounter an email with the header like this?